Missing dependencies

I have forked latest master and when attempting to run mvn clean install I get below error. Looks like dependency was not added in  parent starter , I tried to fix and publish PR but i couldn't find spring-social-microsoft-graph dependency in maven repository. Hence raising the issue.

```
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for com.microsoft.azure:spring-social-microsoft-graph:jar is missing. @ com.microsoft.azure:microsoft-graph-spring-boot-autoconfigure:[unknown-version], C:\Users\rajakolli\git\azure-spring-boot-starters\microsoft-graph\microsoft-graph-spring-boot-autoconfigure\pom.xml, line 61, column 21
[ERROR] 'dependencies.dependency.version' for com.microsoft.azure:microsoft-graph-spring-boot-autoconfigure:jar is missing. @ com.microsoft.azure:microsoft-graph-spring-social-starter:[unknown-version], C:\Users\rajakolli\git\azure-spring-boot-starters\microsoft-graph\microsoft-graph-spring-social-starter\pom.xml, line 46, column 21
[ERROR] 'dependencies.dependency.version' for com.microsoft.azure:microsoft-graph-spring-social-starter:jar is missing. @ line 48, column 21
 @ 
[ERROR] The build could not read 3 projects -> [Help 1]
[ERROR]   
[ERROR]   The project com.microsoft.azure:microsoft-graph-spring-boot-autoconfigure:0.1.4 (C:\Users\rajakolli\git\azure-spring-boot-starters\microsoft-graph\microsoft-graph-spring-boot-autoconfigure\pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for com.microsoft.azure:spring-social-microsoft-graph:jar is missing. @ com.microsoft.azure:microsoft-graph-spring-boot-autoconfigure:[unknown-version], C:\Users\rajakolli\git\azure-spring-boot-starters\microsoft-graph\microsoft-graph-spring-boot-autoconfigure\pom.xml, line 61, column 21
[ERROR]   
[ERROR]   The project com.microsoft.azure:microsoft-graph-spring-social-starter:0.1.4 (C:\Users\rajakolli\git\azure-spring-boot-starters\microsoft-graph\microsoft-graph-spring-social-starter\pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for com.microsoft.azure:microsoft-graph-spring-boot-autoconfigure:jar is missing. @ com.microsoft.azure:microsoft-graph-spring-social-starter:[unknown-version], C:\Users\rajakolli\git\azure-spring-boot-starters\microsoft-graph\microsoft-graph-spring-social-starter\pom.xml, line 46, column 21
[ERROR]   
[ERROR]   The project com.microsoft.azure:microsoft-graph-spring-social-starter-sample:0.0.1-SNAPSHOT (C:\Users\rajakolli\git\azure-spring-boot-starters\microsoft-graph\microsoft-graph-spring-social-starter-sample\pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for com.microsoft.azure:microsoft-graph-spring-social-starter:jar is missing. @ line 48, column 21
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException

```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,7 +2,7 @@ language: java
 jdk:
   - oraclejdk8
 
-script: "mvn cobertura:cobertura"
+script: "mvn clean install"
 
 sudo: required
 cache:

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,12 +2,9 @@ language: java
 jdk:
   - oraclejdk8
 
-script: "mvn clean install"
+script: "mvn cobertura:cobertura"
 
 sudo: required
-cache:
-  directories:
-    - $HOME/.m2
 
 env:
   global:

--- a/common/azure-spring-boot-starter-parent/pom.xml
+++ b/common/azure-spring-boot-starter-parent/pom.xml
@@ -8,7 +8,7 @@
         <groupId>com.microsoft.azure</groupId>
         <artifactId>azure-spring-boot-starter-bom</artifactId>
         <version>0.1.4</version>
-        <relativePath>../pom.xml</relativePath>
+        <relativePath>../../pom.xml</relativePath>
     </parent>
 
     <artifactId>azure-spring-boot-starter-parent</artifactId>

--- a/microsoft-graph/microsoft-graph-spring-boot-autoconfigure/pom.xml
+++ b/microsoft-graph/microsoft-graph-spring-boot-autoconfigure/pom.xml
@@ -5,8 +5,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <artifactId>azure-spring-boot-starter-parent</artifactId>
         <groupId>com.microsoft.azure</groupId>
+        <artifactId>azure-spring-boot-starter-parent</artifactId>
         <version>0.1.4</version>
         <relativePath>../../common/azure-spring-boot-starter-parent/pom.xml</relativePath>
     </parent>

